test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -569,15 +569,9 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				Expect(err).NotTo(HaveOccurred())
 				Expect(len(loadTestPods)).To(Equal(numLoadTestPods))
 
-				By("Waiting 3 minutes for load to take effect")
-				// Wait 3 minutes for autoscaler to respond to load
```

Comment:
Is this not applicable anymore?

Reply:
Correct. We're no longer "waiting", but instead are "continually checking for the desired result", and then "timing out" at some point.